feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt

```diff
@@ -87,7 +89,13 @@ class AboutFragment : Fragment(R.layout.fragment_about), Injectable {
                 aboutItemFactory.create(
                     getString(R.string.about_item_access)
                 ) {
-                    // TODO go access-page
+                    val intent = Intent(
+                        Intent.ACTION_VIEW,
+                        Uri.parse("geo:35.6219252,139.7190626?q=TOCビル")
+                    ).setPackage("com.google.android.apps.maps")
```

Comment:
I want to use implicit intent. So I think this is not needed 👀 
If you have any opinion please tell me.

Reply:
Fixed.